packages/contracts-bedrock/scripts/DeployOwnership.s.sol

```diff
@@ -174,18 +191,43 @@ contract DeployOwnership is Deploy {
         );
     }
 
-    /// @notice Configure the Security Council Safe with the LivenessModule, DeputyGuardianModule, and LivenessGuard.
-    function configureSecurityCouncilSafe() public broadcast returns (address addr_) {
-        // Deploy and add the Deputy Guardian Module.
-        SecurityCouncilConfig memory exampleCouncilConfig = _getExampleCouncilConfig();
-        Safe safe = Safe(mustGetAddress("SecurityCouncilSafe"));
+    /// @notice Deploy Guardian Safe.
+    function deployGuardianSafe() public broadcast returns (address addr_) {
+        // Config is hardcoded here as the Guardian Safe's configuration is inflexible.
+        address[] memory owners = new address[](1);
+        owners[0] = mustGetAddress("SecurityCouncilSafe");
+        addr_ = deploySafe({ _name: "GuardianSafe", _owners: owners, _threshold: 1, _keepDeployer: true });
+
+        console.log("Deployed and configured the Guardian Safe!");
+    }
+
+    /// @notice Configure the Guardian Safe with the DeputyGuardianModule.
+    function configureGuardianSafe() public broadcast returns (address addr_) {
+        Safe safe = Safe(payable(mustGetAddress("GuardianSafe")));
         address deputyGuardianModule = deployDeputyGuardianModule();
         _callViaSafe({
             _safe: safe,
             _target: address(safe),
             _data: abi.encodeCall(ModuleManager.enableModule, (deputyGuardianModule))
         });
-        console.log("DeputyGuardianModule enabled on SecurityCouncilSafe");
+
+        // Remove the deployer address (msg.sender) which was used to setup the Security Council Safe thus far
+        // this call is also used to update the threshold.
+        // Because deploySafe() always adds msg.sender first (if keepDeployer is true), we know that the previousOwner
+        // will be SENTINEL_OWNERS.
+        _callViaSafe({
+            _safe: safe,
+            _target: address(safe),
+            _data: abi.encodeCall(OwnerManager.removeOwner, (SENTINEL_OWNERS, msg.sender, 1))
+        });
+        console.log("DeputyGuardianModule enabled on GuardianSafe");
```

Comment:
Alternatively, you can atomically deploy the Safe + install the DGM by passing `to` and `data` to simplify this flow: https://github.com/safe-global/safe-smart-account/blob/5feb0d08f59cfbb44918be1ed5889d9bb634562a/contracts/Safe.sol#L95

Reply:
I'd prefer not to: 
1. It feels like that will create a fair bit of additional gymnastics: 
    1. The DGM needs to know the Safe's address when it is deployed.
    2. `setupModules()` uses a `DelegateCall`, so I think some temporary logic contract is needed that writes to the correct storage slots to insert the module into the linked list

2. The GuardianSafe is already deployed on Sepolia, and will already be deployed on Mainnet (and we prefer using the UI to deploy). So this is more similar to what we'll be doing and is more easily edited for the modules/guard deployments.